docs/helm/helm_template.md

```diff
@@ -26,7 +26,7 @@ helm template [flags] CHART
 
 ```
   -x, --execute stringArray    only execute the given templates
-      --kube-version string    override the Kubernetes version used as Capabilities.KubeVersion.Major/Minor (e.g. 1.7)
+      --kube-version string    kubernetes version used as Capabilities.KubeVersion.Major/Minor (default "v1.9.0")
```

Comment:
Can we keep this in the form "1.9"? The major and minor version fields are the only fields we change using --kube-version so `v` and `.0` are superfluous.

Reply:
I fixed it at 67758a4.